.github/workflows/build.yml

```diff
@@ -77,14 +77,16 @@ jobs:
         run: |
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config user.name "github-actions[bot]"
-      - name: No version bump
+      #   -- Version Bumping --
+      - name: Manual execution means no version bump
         # If triggered by workflow dispatch, no version bump
         if: ${{ github.event_name == 'workflow_dispatch' }}
         run: |
           echo "TARGET_ENV=${{ github.event.inputs.venue }}" >> $GITHUB_ENV
```

Comment:
I'm not sure what this is meant to do if the bumping is already guarded by the `&& github.event_name != 'workflow_dispatch'` below

Reply:
This section sets the ENV value so that we can deploy to the proper environment, either SIT or UAT.  The protected steps below set that value based on the branch, but without this step it will be empty during workflow_dispatch runs which breaks the deployment